client/components/select-dropdown/style.scss

```diff
@@ -40,7 +40,7 @@ $compact-header-height: 35;
 	}
 
 	.accessible-focus & {
-		border-radius: 2px;
+		border-radius: 4px;
```

Comment:
I realize that any changes in this file will apply to most `SelectDropdown`s in wp-calypso. I think it just adds the more modern touch to the component (it's 7 years old) and aligns closer with our recent designs. However if one reviewing would prefer I would only apply those for the accent color select-dropdown, I will separate this out.